### PR TITLE
feat: Use kubo v0.15.0 instead of go-ipfs

### PR DESCRIPTION
**Pull request by ukstv**
Migrate from "go-ipfs" to "kubo" v0.15.0.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,9 +21,9 @@ COPY --from=clone /clone/go-ipfs/go.mod /clone/go-ipfs/go.sum $SRC_DIR/
 COPY --from=clone /clone/go-ipfs $SRC_DIR
 
 RUN cd $SRC_DIR \
-  && go get github.com/ceramicnetwork/go-ipfs-healthcheck/plugin@v0.11.0 \
-  && go get github.com/3box/go-ds-s3/plugin@v0.11.0 \
-  && go get github.com/cheggaaa/pb@v1.0.29
+  && go get github.com/ceramicnetwork/go-ipfs-healthcheck/plugin@v0.14.0 \
+  && go get github.com/3box/go-ds-s3/plugin@v0.14.0
+#  && go get github.com/cheggaaa/pb@v1.0.29
```
Comment (v-stickykeys) [inline Dockerfile:26]:
```suggestion
```

Should we just delete this line?

**Review by smrz2001 — APPROVED** [review]
🚀

**Comment by ukstv** [comment]
@smrz2001 What is wrong with the GitHub action here, do you know?

**Comment by stbrody** [comment]
Looks like 0.15 just got released a few hours ago: https://github.com/ipfs/kubo/releases/tag/v0.15.0

Should we see if it's easy to go all the way to 15 while we're at it?

**Comment by 3avi** [comment]
@stbrody any risks of going straight v0.15? v0.14 might be more stable as we head into GR15 launch?

**Comment by ukstv** [comment]
Now it is v0.15.0

**Pull request merged**